docs_2.0/manual_EN/3.quick-start/4.nebula-graph-crud.md

```diff
@@ -75,14 +75,14 @@ From the **Status** column of the table in the return message, you can see that
 1. Use the following statement to create a graph space named `nba`.
 
     ```nGQL
-    (root@nebula) [(none)]> CREATE SPACE nba(partition_num=15, replica_factor=1);
+    nebula> CREATE SPACE nba(partition_num=15, replica_factor=1, vid_type=fixed_string(30));
```

Comment:
you should add some explanation about the 30. 
is it similar to mysql char(N) varchar(N) or others?
it is worth to add a .md file to explain the details of fixed_string.

Reply:
Details are supposed to be in the `create space` doc. I'll add some instructions here.